backend/src/data/repositories/mentees-to-mentors/mentees-to-mentors.repository.ts

```diff
@@ -30,15 +30,16 @@ class MenteesToMentors {
   public async getByCourseIdAndMenteeId(getMenteesToMentors: {
```

Comment:
As for now this method won't return any record - so maybe it makes sense to name it properly